### PR TITLE
add more guides

### PR DESCRIPTION
**Pull request by Mineinjava**

### CODE REVIEW

**Comment by Mineinjava** [comment]
@Astr0clad anything else you want here?

File: guides/math.md [hunk 1]
```diff
@@ -6,70 +6,99 @@ icon: code-square
...
+rather complex.
 
 coming soon™
 
-## Forwards Kinematics (Differential Swerve)
-***Please note that in some differential swerve setups one motor's rotation is reversed, which will require inversion of its veocity and position***
+## Forwards Kinematics (sifferential swerve per-module)
+
+!!!secondary
+Please note that in some differential swerve setups one motor's rotation is reversed, which will require inversion of its veocity and position
```
Comment (cmontigel) [inline guides/math.md]: Velocity spelling

File: localization/SwerveOdo.md [hunk 2]
```diff
@@ -7,16 +7,42 @@ icon: issue-draft
...
 ### Class Summary
+
 ```java
 swerveOdometry(Vec2d[] moduleVectors)
 swerveOdometry(List<Vec2d> moduleVectors)
 swerveOdometry(swerveDrive drivetrain)
 ```
-You should probably give it what it wants. Using the swerveDrive constructor is recommended.
+
+Instntiate a new `swerveOdometry` object with the module positions of your swerve drive. You can also pass in a `swerveDrive` object to get the position vectors from that.
```
Comment (cmontigel) [inline localization/SwerveOdo.md]: Spelling of Instantiate

**Review by cmontigel — COMMENTED** [review]
Everything looks good other than the two spelling issues, I'll resolve them and then approve

**Review by cmontigel — APPROVED** [review]
Looks great :D

**Pull request merged**